src/revlanguage/workspace/FunctionTable.cpp

```diff
@@ -314,14 +314,24 @@ const Function* FunctionTable::findFunction(const std::string& name, const std::
     ret_val = equal_range(name);
     if (hits == 1)
     {
-        if (ret_val.first->second->checkArguments(args,NULL,once) == false)
+        std::vector<bool> arg_mapped(args.size(), false);
+        if (ret_val.first->second->checkArguments(args,NULL,arg_mapped,once) == false)
```

Comment:
It looks like `arg_mapped` is basically a return value only, and not an input value... is that right?
In theory I guess you could return a pair<bool, vector<bool>>, but that could be cumbersome.